### PR TITLE
Create CHANGELOG.rst

### PR DESCRIPTION
**Pull request by cybernth**

### CODE REVIEW

**Comment by r0h4n** [comment]
the file type should be .rst

**Review by r0h4n — CHANGES_REQUESTED** [review]
file should be .rst and not .md

**Comment by cybernth** [comment]
@r0h4n changed the format of the file

File: doc/source/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+###########
+Change Log
+###########
+
+
+All notable changes to this project will be documented in this file.
+
+*************
+[Unreleased]
```
Comment (shtripat) [inline doc/source/CHANGELOG.md:9]: Released now?

File: doc/source/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- Added the basic project infrastructure
+- Integrated the ci and code coverage tools
+- Added the unit test infrastructure
+- Added the etcd communication infrastructure
+- Added the job execution framework
+- Support for import ceph cluster
+- Support for pool CRUD opereations
+- User documentation and README
+
+[Unreleased]: <<github tag for this release>>
```
Comment (shtripat) [inline doc/source/CHANGELOG.md:23]: Released and add the correct tag name

**Comment by ktdreyer** [comment]
Git commit logs still say `.md` rather than `.rst`. Want to amend and squash down?

In ceph-installer we've tried to follow the basic format that the [gitchangelog](https://pypi.python.org/pypi/gitchangelog) tool generates. it tends to speed things up when writing the text.

**Pull request closed without merging**